cookbooks/bcpc-hadoop/libraries/find_object_by_group.rb

```diff
@@ -0,0 +1,63 @@
+
+module Bcpc_Hadoop
+  module Helper
+
+    include Chef::Mixin::ShellOut
+
+    # Initialize LDAP
+    #
+    # keytab - File path location of keytab to use for GSSAPI authentication
+    #
+    # Post-Condition: Sets necessary module constants needed by
+    #                 find_object_by_group()
+    def initialize_ldap(keytab, ldap_host)
```

Comment:
I would like a trivial mock LDAP server for integration tests. Let me see if I can get this to work for password auth in a full environment too. For now, let us move forward with kerberized LDAP and look how to do bind password auth in the future.